1-exercises/C-highlight-words/script.js

```diff
@@ -1,7 +1,38 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  let content = document.getElementById("content");
+  let select = document.createElement("select");
+  content.appendChild(select);
+  let p = document.createElement("p");
+  content.appendChild(p);
+
+  for(colour of colours){
+    let option = document.createElement("option");
+    select.appendChild(option);
+    let optionText = document.createTextNode(colour);
+    option.appendChild(optionText);
+  }
+
+  let newArray = paragraph.split(" ");
```

Comment:
Try to replace the newArray variable name with a new name which represents what it contains. Maybe include "word" in the name as it's an array full of words rather than something else like ints.

We can find out the type that split returns is a newArray from the docs or using typeof on the variable but difficult to find the context the variable is used in without reading more code.